web/src/auth/ProviderSignupPage.js

```diff
@@ -0,0 +1,440 @@
+// Copyright 2021 The casbin Authors. All Rights Reserved.
```

Comment:
There are some bugs in this page, like this:
![image](https://user-images.githubusercontent.com/42638489/121809219-8abd0d80-cc8e-11eb-9fa2-625e1b2faf29.png)

And we already have a signup page, how about reusing the page here?

Reply:
This cause I didn't set email and SMS verification server, already working on it.
Reusing the page may transfer too many params, is that ok?